lib/main.dart

```diff
@@ -42,7 +42,7 @@ class MyHomePage extends StatefulWidget {
 class _MyHomePageState extends State<MyHomePage> {
   Game _game = _createGame(initialGridName);
   final ValueNotifier<String> _grid = ValueNotifier<String>(initialGridName);
-  late ValueNotifier<Coordinates> _lastMove;
+  final ValueNotifier<Coordinates?> _lastMove = ValueNotifier(null);
```

Comment:
What's the point to have _lastMove here?
May be it should be owned by Game?

Reply:
Game owns it, but I need it here as well, otherwise I can't catch when it's being changed (game object doesn't change when lastMove does)